Services/Music_Portal.Services.Services/LastFmService.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
-using Interfaces;
-using Interfaces.Models;
+using Services.Interfaces.Models;
+using Services.Interfaces;
 
-namespace Services
+namespace Services.Services
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Services.Services`